xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
+    This allows to associate &rootuser; logins with particular users.
+    This also adds another layer of security, because not only the root password
+    but also the password of an administrator's account needs to be compromised
+    to gain full root access. This section explains how to limit direct root
+    logins on the different levels of the system.
+   </para>
+   <sect2 xml:id="sec-sec-prot-restrict-root-tty">
+    <title>Restricting Local Text Console Logins</title>
+
+    <para>
+     TTY devices provide text mode system access via the console. For desktop
+     systems these are accessed via the local keyboard or in case of server
+     systems via input devices connected to a KVM switch or a remote
+     management card (ILO, DRAC, etc).
+     By default, Linux offers 6 different consoles, that can be switched to
+     via the key-combinations
+     <keycombo><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="alt"/><keycap>F6</keycap></keycombo>, when
+     running in text mode, or
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F6</keycap></keycombo>
+      when running in a graphical session. The
+      associated terminal devices are named <literal>tty1</literal> .. <literal>tty6</literal> accordingly.
+    </para>
+
+    <para>
+     The following steps restrict root access to the first TTY. Even this
+     access method is only meant for emergency access to the system and should
+     never be used for everyday system administration tasks.
+    </para>
+
+    <itemizedlist>
+     <listitem>
+     <para>
+      Ensure that the PAM stack configuration file <filename>/etc/pam.d/login</filename>
+      contains the <literal>pam_securetty</literal> module in the <literal>auth</literal> block:
+     </para>
+     <screen>auth     requisite      pam_nologin.so
+auth     [user_unknown=ignore success=ok ignore=ignore auth_err=die default=bad] pam_securetty.so noconsole
+auth     include        common-auth</screen>
+     </listitem>
+     <listitem>
+      <para>
+       This will include the <literal>pam_securetty</literal> PAM module during the
+       authentication process on local consoles to allow root only to login on
+       tty devices that are listed in the file <filename>/etc/securetty</filename>.
+       Thus to actually limit the tty devices for root you need to remove all
+       entries from this file except one:
+      </para>
+     <screen>#
+# This file contains the device names of tty lines (one per line,
+# without leading /dev/) on which root is allowed to login.
+#
+tty1</screen>
+     </listitem>
+     <listitem>
+      <para>
+       After these changes root logins from non-allowed terminals will be
+       rejected right away without even querying the root password.
```

Comment:
I'm not sure about this. What should the user do here? It sounds like it's pure information? If yes, then it should be added after the action. Maybe add it to the last step after the `<screen>`?. 

I would create a different action here. Something like "Check bla..." to see if everything is okay. Does that make sense?

Reply:
Makes sense. I'll change it into a verification step.